IdentityCore/src/util/MSIDJsonObject.m

```diff
@@ -26,6 +26,7 @@
 //------------------------------------------------------------------------------
 
 #import "MSIDJsonObject.h"
+#import "MSIDError.h"
```

Comment:
MSIDError is part of the prefix file. Why does it need to be imported separately here?

Reply:
The additional #imports have to do with building without the .pch file.

Reply:
Why are we building without the pch?

Reply:
I was getting build errors for our build, but I'll try again getting the pch working.